akka-service-locator/core/src/main/resources/reference.conf

```diff
@@ -1,3 +1,4 @@
+# #lagom-akka-discovery-reference-conf
```

Comment:
This is a starting anchor but you will need to add a closing one (just copy the anchor line where you want to snippet to finish.

Reply:
Done. Sorry missed that.